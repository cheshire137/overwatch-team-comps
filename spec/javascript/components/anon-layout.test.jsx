import renderer from 'react-test-renderer'

import AnonLayout from '../../../app/assets/javascripts/components/anon-layout.jsx'

describe('AnonLayout', () => {
  let component = null

  beforeEach(() => {
    const location = { pathname: '/about' }
    component = (
      <AnonLayout location={location}><p>Hey there</p></AnonLayout>
    )
  })

  test('matches snapshot', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
