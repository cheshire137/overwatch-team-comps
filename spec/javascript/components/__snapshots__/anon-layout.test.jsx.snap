exports[`AnonLayout matches snapshot 1`] = `
<div
  className="layout-container">
  <div
    className="container">
    <nav
      className="nav">
      <div
        className="nav-right">
        <a
          className="nav-item"
          href="/">
          Team composition
        </a>
        <a
          className="nav-item"
          href="/users/auth/bnet">
          Sign in with Battle.net
        </a>
      </div>
    </nav>
  </div>
  <div
    className="layout-children-container">
    <p>
      Hey there
    </p>
  </div>
  <footer
    className="footer">
    <div
      className="container">
      <a
        href="/about">
        About
      </a>
      <a
        href="https://github.com/cheshire137/overwatch-team-comps"
        rel="noopener noreferrer"
        target="_blank">
        View source
      </a>
    </div>
  </footer>
</div>
`;
